Modules/System/Language/src/Language.Codeunit.al

```diff
@@ -152,6 +152,17 @@ codeunit 43 Language
         LanguageImpl.LookupWindowsLanguageId(LanguageId);
     end;
 
+    /// <summary>
+    /// Returns an alternative language based on Windows Culture Info.
```

Comment:
It's not an alternative language, it's the culture identifier of the parent language, isn't it?

Reply:
This is what is on MSDN. Maybe we should copy/paste.
Gets the CultureInfo that represents the parent culture of the current CultureInfo.

Reply:
Changed